study_time/courses/tests/fixtures.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from courses.tests import values
+from .factories import CourseFactory, EpisodeFactory
+
+
+@pytest.fixture
+def course_fixture():
+    return CourseFactory(id=values.COURSE_ID)
```

Comment:
```suggestion
def course_fixture():
    return CourseFactory(id=values.COURSE_ID,
        title=values.COURSE_TITLE,
        video_duration= values.COURSE_VIDEO_DURATION,
        language=values.COURSE_LANGUAGE,
    )
```
It's good to have fixtures produce as consistent results as possible